ml4gw/waveforms/cbc/phenom_d.py

```diff
@@ -384,17 +407,20 @@ def phenom_d_inspiral_amp(
 
         return amp, Damp
 
-    def phenom_d_phase(self, Mf, mass_1, mass_2, eta, eta2, chi1, chi2, xi):
+    def phenom_d_phase(
+        self, Mf, mass_1, mass_2, eta, eta2, chi1, chi2, xi, fRD=None, fDM=None
```

Comment:
Looks good. Just leave a comment here that the `None` values are for phenomd and for phenomp the value need to be supplied.